app/views/issues/show.html.erb

```diff
@@ -1,3 +1,9 @@
+<div>
+  <%= link_to @issue.series do %>
+    <%= @issue.series_id %>
```

Comment:
`<%= link_to @issue.series.title, series_path(@issue.series) %>` I think should do this in one line.

Reply:
It works like `link_to(text_of_link, destination_of_link, other_optional_stuff)` the `series_path` helper comes from the routes file. You can see all the `<whatever>_path` helpers by typing `rails routes` in your terminal (not console)